<launch>

  <node name="image_viewer" pkg="rab_bringup" type="image_viewer" output="screen">
    <param name="odom_topic" value="/diff_drive_controller/odom" />
    <param name="first_path" value="$(find rab_bringup)/picture/nobori.jpg" />
    <param name="first_x" value="5.0" />
    <param name="first_y" value="0.0" />
    <param name="second_path" value="$(find rab_bringup)/picture/lab_caution.jpg" />
    <param name="second_x" value="-5.0" />
    <param name="second_y" value="0.0" />
    <param name="default_path" value="$(find rab_bringup)/picture/sample.JPG" />
  </node>
  
</launch>
